handlers/events_streamer.go

```diff
@@ -57,14 +57,17 @@ func (h *Handler) EventStreamHandler(w http.ResponseWriter, req *http.Request, _
 	// defer close(newAdaptersChan)
 
 	go func() {
+
 		for mClient := range newAdaptersChan {
-			log.Debug("received a new mesh client, listening for events")
-			go func() {
-				listenForAdapterEvents(req.Context(), mClient, respChan, log)
-				_ = mClient.Close()
-			}()
+      		log.Debug("received a new mesh client, listening for events")
+      		go func(mClient *meshes.MeshClient) {
```

Comment:
Is there any idea about how to test this function?

Reply:
@Aisuko I didn't quite get you here. Are you asking about the functionality test of the function or how will write unit tests for this function ?

Reply:
If you are concerned about the functionality, I have tested the function and output is same as before.

Reply:
@kushthedude This would be awesome, good job. Thanks